pkg/test/framework/runtime/components/pilot/native.go

```diff
@@ -90,9 +101,11 @@ func (c *nativeComponent) Start(ctx context.Instance, scope lifecycle.Scope) (er
 	}
 
 	bootstrapArgs := bootstrap.PilotArgs{
-		Namespace:        env.Namespace,
-		DiscoveryOptions: options,
-		MeshConfig:       env.Mesh,
+		Namespace:         env.Namespace,
+		DiscoveryOptions:  options,
+		MeshConfig:        env.Mesh,
+		MCPServerAddrs:    []string{galley.GetGalleyAddress()},
```

Comment:
Just curious - why aren't these MCP params part of the MeshConfig?

Reply:
Will use meshConfig as seems MCPServerAddrs will be depreciated.